src/app/query/tokens/fungible-token-metadata.query.ts

```diff
@@ -1,21 +1,34 @@
 import { useQueries, useQuery } from 'react-query';
+import { RateLimiter } from 'limiter';
 
 import { useApi, Api } from '@app/store/common/api-clients.hooks';
 import { useCurrentNetwork } from '@app/common/hooks/use-current-network';
+import { isResponseCode } from '@app/common/network/is-response-code';
 
-const staleTime = 10 * 60 * 1000;
+const limiter = new RateLimiter({ tokensPerInterval: 1, interval: 750 });
+
+const staleTime = 20 * 60 * 1000;
 
 const queryOptions = {
   keepPreviousData: true,
   cacheTime: staleTime,
+  staleTime: staleTime,
   refetchOnMount: false,
   refetchInterval: false,
   refetchOnReconnect: false,
-  retryDelay: 120_000,
+  retry: 2,
+  retryDelay: 2 * 60 * 1000,
```

Comment:
Should this be moved to a constant too?

Reply:
this value isn't reused, and don't reckon it'd add much in way of readability